### ISSUE TITLE
Add a "Text size" setting to the Settings / Appearance page

### ISSUE THREAD

**Issue by davidmz** [FreeFeed/freefeed-react-client#1358]
See https://freefeed.net/support/61a89352-0964-4f1e-b38a-27d9846ae7d6

**Comment by dsumin** [comment]
Released!

**Comment by tcharge** [comment]
I have issues with this. Firefox 93.0.
After changing the setting, I got a confirmation message but then, when I try to leave the Settings page, I receive a system notification that all changes made on the page will be lost (and they are!)
So the text size setting doesn't work for me.

**Comment by kukutz** [comment]
@tcharge did you try to save settings before leaving settings page?

**Comment by tcharge** [comment]
@kukutz, Sure I did (pressed a button near the bottom of page, and have received a green confirmation message).
But afterwards, when clicking any other navigational links, I'm getting a system warning on losing all changes I've made.

**Comment by davidmz** [comment]
У меня не получается воспроизвести. Можно более подробный сценарий? Ты заходишь на вкладку, меняешь размер шрифта, нажимаешь внизу «Применить»? Это всё не в инкогнито-режиме?

**Comment by tcharge** [comment]
Да, двигаю слайдер влево до 120%, внизу нажимаю "Update appearance",  получаю алерт "Appearance updated".
Поднимаюсь вверх страницы, нажимаю, например, на вкладку "Privacy". Получаю системный алерт "Do you want to leave this page? Changes you made may not be saved."
Соответственно, размер шрифта не меняется.

**Comment by davidmz** [comment]
А что говорит `localStorage.getItem("ui-scale")` в консоли? И в какой OS дело происходит?

**Comment by kukutz** [comment]
Я попробовал воспроизвести, у меня выглядит так:

1. двигаю слайдер влево до 120%
2. внизу нажимаю "Update appearance"
3. получаю алерт "Appearance updated" **и изменение шрифта**
4. Поднимаюсь вверх страницы, нажимаю, например, на вкладку "Privacy". 
5. Получаю системный алерт "Do you want to leave this page? Changes you made may not be saved."
6. Соответственно, размер шрифта меняется.

(OS Windows, Firefox)

**Comment by tcharge** [comment]
Windows 10.
OK now it seems to be working (text size is changing). Yet there are two bugs remaining:
1) the above-mentioned system alert after saving changes.
2) the font size slider position in 'Settings' is back to default, when the page is refreshed or visited again. Though the value in localStorage is correct and updated.
